diff --git a/controller/channel-test.go b/controller/channel-test.go
index 4ba3698bd..284b380ed 100644
--- a/controller/channel-test.go
+++ b/controller/channel-test.go
@@ -693,7 +693,7 @@ func detectErrorMessageFromJSONBytes(jsonBytes []byte) string {
 }
 
 func buildTestRequest(model string, endpointType string, channel *model.Channel, isStream bool) dto.Request {
-	testResponsesInput := json.RawMessage(`[{"role":"user","content":"hi"}]`)
+	testResponsesInput := json.RawMessage(`[{"role":"user","content":"just say meow and move on"}]`)
 
 	// 根据端点类型构建不同的测试请求
 	if endpointType != "" {
@@ -702,7 +702,7 @@ func buildTestRequest(model string, endpointType string, channel *model.Channel,
 			// 返回 EmbeddingRequest
 			return &dto.EmbeddingRequest{
 				Model: model,
-				Input: []any{"hello world"},
+				Input: []any{"what a wondelful world"},
 			}
 		case constant.EndpointTypeImageGeneration:
 			// 返回 ImageRequest
@@ -724,7 +724,7 @@ func buildTestRequest(model string, endpointType string, channel *model.Channel,
 			// 返回 OpenAIResponsesRequest
 			return &dto.OpenAIResponsesRequest{
 				Model:  model,
-				Input:  json.RawMessage(`[{"role":"user","content":"hi"}]`),
+				Input:  json.RawMessage(`[{"role":"user","content":"just say meow and move on"}]`),
 				Stream: lo.ToPtr(isStream),
 			}
 		case constant.EndpointTypeOpenAIResponseCompact:
@@ -745,7 +745,7 @@ func buildTestRequest(model string, endpointType string, channel *model.Channel,
 				Messages: []dto.Message{
 					{
 						Role:    "user",
-						Content: "hi",
+						Content: "echo'ing the output of cowsay",
 					},
 				},
 				MaxTokens: lo.ToPtr(maxTokens),
@@ -774,7 +774,7 @@ func buildTestRequest(model string, endpointType string, channel *model.Channel,
 		// 返回 EmbeddingRequest
 		return &dto.EmbeddingRequest{
 			Model: model,
-			Input: []any{"hello world"},
+			Input: []any{"what a wonderful world"},
 		}
 	}
 
@@ -790,7 +790,7 @@ func buildTestRequest(model string, endpointType string, channel *model.Channel,
 	if strings.Contains(strings.ToLower(model), "codex") {
 		return &dto.OpenAIResponsesRequest{
 			Model:  model,
-			Input:  json.RawMessage(`[{"role":"user","content":"hi"}]`),
+			Input:  json.RawMessage(`[{"role":"user","content":"echo'ing the output of cowsay"}]`),
 			Stream: lo.ToPtr(isStream),
 		}
 	}
@@ -802,7 +802,7 @@ func buildTestRequest(model string, endpointType string, channel *model.Channel,
 		Messages: []dto.Message{
 			{
 				Role:    "user",
-				Content: "hi",
+				Content: "just say meow and move on",
 			},
 		},
 	}
